src/js/taos.js

```diff
@@ -50,7 +50,7 @@
 
   const refresh = () => {
     elements = []
-    document.querySelectorAll('[class*="taos"]').forEach(el => elements.push(initElement(el)))
+    document.querySelectorAll('[class*="Taos"]').forEach(el => elements.push(initElement(el)))
```

Comment:
```suggestion
    document.querySelectorAll('[class*="taos"]').forEach(el => elements.push(initElement(el)))
```

online editor upcased this 🤨